frontend/src/components/App/Home/__snapshots__/SquareButton.stories.storyshot

```diff
@@ -8,14 +8,14 @@ exports[`Storyshots common/SquareButton Basic 1`] = `
     type="button"
   >
     <div
-      class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root makeStyles-root makeStyles-root css-bhp9pd-MuiPaper-root-MuiCard-root"
+      class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-1o0qjyj-MuiPaper-root-MuiCard-root"
```

Comment:
Will these names be generated on each CI run?? If yes then that can be an issue

Reply:
They seems to be static.